auth.js

```diff
@@ -24,16 +24,27 @@ passport.use(new LocalStrategy(
         return done(null, false, { message: 'Incorrect username.' })
       }
       // Sprawdź poprawność hasła
-      bcrypt.compare(password, user.password, function(err, res) {
-        if (!res) {
-          return done(null, false, { message: 'Incorrect password.' })
-        } else if (!user.approved) {
-          return done(null, false, { message: 'You have been banned.' })
-        } else {
-          // Zalogowano poprawnie, zwróć obiekt zalogowanego użytkownika
-          return done(null, user, { message: 'Welcome!' })
-        }
-      })
+      if(config.service == 'rethinkdb') {
```

Comment:
W kodzie produkcyjnym nie powinno być sprawdzania takich warunków, ani żadbych innych odwołań do środowiska testowego, gdyż niepotrzebnie komplikuje to kod i wprowadza potencjalne problemy. Jeżeli w bazie danych kodujemy hasła to w plikach też to róbmy.